osu.Game.Rulesets.Mania/UI/ManiaPlayfield.cs

```diff
@@ -157,82 +99,45 @@ public ManiaPlayfield(int columnCount)
         private void invertedChanged(bool newValue)
         {
             Scale = new Vector2(1, newValue ? -1 : 1);
-            judgements.Scale = Scale;
-        }
-
-        [BackgroundDependencyLoader]
-        private void load(OsuColour colours)
-        {
-            normalColumnColours = new List<Color4>
-            {
-                colours.RedDark,
-                colours.GreenDark
-            };
-
-            specialColumnColour = colours.BlueDark;
-
-            // Set the special column + colour + key
-            foreach (var column in Columns)
-            {
-                if (!column.IsSpecial)
-                    continue;
 
-                column.AccentColour = specialColumnColour;
-            }
-
-            var nonSpecialColumns = Columns.Where(c => !c.IsSpecial).ToList();
-
-            // We'll set the colours of the non-special columns in a separate loop, because the non-special
-            // column colours are mirrored across their centre and special styles mess with this
-            for (int i = 0; i < Math.Ceiling(nonSpecialColumns.Count / 2f); i++)
+            foreach (var single in listColumnStages)
             {
-                Color4 colour = normalColumnColours[i % normalColumnColours.Count];
-                nonSpecialColumns[i].AccentColour = colour;
-                nonSpecialColumns[nonSpecialColumns.Count - 1 - i].AccentColour = colour;
+                single.Judgements.Scale = Scale;

```

Comment:
How about giving the stages an `Inverted` BindableBool, and having them decide their scale?